templates/emails/collective.virtualcard.missing.receipts.hbs

```diff
@@ -0,0 +1,27 @@
+Subject: {{collective.name}}'s {{virtualCard.last4}} card is missing receipt(s)
+
+
+{{> header}}
+
+<center>
+  <h1>
+    {{collective.name}}'s {{virtualCard.name}} ({{virtualCard.last4}}) card is missing receipt(s).
+  </h1>
+</center>
+
+<p>
+  Avoid having your card suspended by providing the receipt for the following charges within {{daysLeft}} days:
```

Comment:
Add a sentence preceding this line: "Per your fiscal host's virtual card policy, receipts must be uploaded for virtual card expenses within 31 days."

Reply:
I don't think this is necessarily true for all hosts.
Any way we can make this more generic?